### ISSUE TITLE
1.1.2 Update Error

### ISSUE THREAD

**Issue by MurhafSousli** [MurhafSousli/ng2-sharebuttons#22]
@tinesoft After pushing the updates to npm in 1.1.2, I get this error

```
ERROR in Error encountered resolving symbol values statically. Calling function 'makeDecorator', function calls are not supporte
d. Consider replacing the function or lambda with a reference to an exported function, resolving symbol NgModule in C:/Users/Mur
haf/Documents/GitHub/ng2-sharebuttons/demo/node_modules/ng2-sharebuttons/node_modules/@angular/core/src/metadata/ng_module.d.ts,
 resolving symbol NgModule in C:/Users/Murhaf/Documents/GitHub/ng2-sharebuttons/demo/node_modules/ng2-sharebuttons/node_modules/
@angular/core/src/metadata.d.ts, resolving symbol NgModule in C:/Users/Murhaf/Documents/GitHub/ng2-sharebuttons/demo/node_module
s/ng2-sharebuttons/node_modules/@angular/core/src/core.d.ts, resolving symbol NgModule in C:/Users/Murhaf/Documents/GitHub/ng2-s
harebuttons/demo/node_modules/ng2-sharebuttons/node_modules/@angular/core/index.d.ts, resolving symbol ShareButtonsModule in C:/
Users/Murhaf/Documents/GitHub/ng2-sharebuttons/demo/node_modules/ng2-sharebuttons/dist/share-buttons.module.d.ts, resolving symb
ol ShareButtonsModule in C:/Users/Murhaf/Documents/GitHub/ng2-sharebuttons/demo/node_modules/ng2-sharebuttons/dist/share-buttons
.module.d.ts
```
I also want to mention that we should add `index.js` to the root because now importing is only possible from `dist` dir 
`import {ShareButtonsModule} from 'ng2-sharebuttons/dist`

**Comment by tinesoft** [comment]
Hi @MurhafSousli ,

I have downloaded latest v1.1.2 and looked inside the installed package in my `node_modules/ng2-sharebuttons` folder. The problem is that it still contains the whole project structure, where as it should only contains what is inside 'dist' folder (conf. PR#19).

Did you published using the new  gulp task i've added in PR#19: `gulp publish` ?
Which prepare the 'dist' folder and them run `npm publish`  on that folder.

You can also run `gulp build` and them `npm publish ./dist`

**Comment by MurhafSousli** [comment]
@tinesoft This can be solved by excluding directories in [`npmignore`](https://github.com/MurhafSousli/ng2-sharebuttons/blob/master/.npmignore)

yes I used `gulp publish`, the problem only occurs when I push it to npm, locally after compiling I tried to use in the demo and it works, I don't know the cause.

**Comment by tinesoft** [comment]
Hi, 

you mean :  `npm run prepublish`? (not `gulp run prepublish`)
If so, this command runs the **default gulp** task (see `"scripts"` section of `package.json`),
which build the `dist `folder.

Could you try my second proposition:

`gulp build`

 then

`npm publish ./dist`   from project root

I believed we shouldn't have to specify an `.npmignore` file because only the `dist `folder is supposed to be published to NPM, and it already contains just everything needed for the package

**Comment by MurhafSousli** [comment]
@tinesoft sorry I meant `npm run prepublish`, okay I will try

**Comment by MurhafSousli** [comment]
Fixed in `1.1.3`
